bears/c_languages/CPPCheckBear.py

```diff
@@ -0,0 +1,35 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable="cppcheck",
+        use_stdout=False,
+        use_stderr=True,
+        output_format="regex",
+        output_regex=r'(?P<line>\d+):(?P<severity>[a-zA-Z]+):'
+                     r'(?P<origin>[a-zA-Z]+):(?P<message>.*)',
+        severity_map={"error": RESULT_SEVERITY.MAJOR,
+                      "warning": RESULT_SEVERITY.NORMAL,
+                      "style": RESULT_SEVERITY.INFO})
+class CPPCheckBear:
+    """
+    Report possible security weaknesses for C/C++.
+    For more information, consult <https://github.com/danmar/cppcheck>.
+    """
+
+    LANGUAGES = "C", "C++"
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         enable_all_checks: bool=True):
+        """
+        :param enable_all_checks:
+            Report warnings and style issues too.
+        """
+        args = ("--template={line}:{severity}:{id}:{message}",)
+
+        if enable_all_checks:
+            args += ('--enable=all',)
```

Comment:
All options can be found using cppcheck -h
but the problem I encounter is with the cppcheck versions. Basically different versions clasify the same issue with different severity (warn gets confused with style).
Also even though there are more options the error, warning, style, I couldn't find a piece of code that would trigger those.